app/models/immunisation_import.rb

```diff
@@ -177,4 +186,16 @@ def rows_are_valid
       end
     end
   end
+
+  def counts_are_nil_or_present
+    if processed?
+      COUNT_COLUMNS.each do |column|
+        errors.add(column, :blank) if send(column).nil?
+      end
+    else
+      COUNT_COLUMNS.each do |column|
```

Comment:
When would this path ever be taken? Do we really need to account for it?

Reply:
Yeah I was in two minds about whether it was worth doing, I put it in because it is a requirement for the stats to have been calculated if it's processed, but you're right that there's only one code paths which processes these. I figured there was no harm in having it in there, and best case it'll spot perhaps a manual update to the records in the future.

Reply:
Would it make sense to use something that would raise here? That would give us an alert in Sentry which might be a clearer indication of our intention here.

But ¯\\\_(ツ)_/¯ ... :shipit: first